generators/templates/package.json

```diff
@@ -21,6 +21,10 @@
   },
   "main": "<%= elementName %>.umd.js",
   "module": "<%= elementName %>.js",
+  "files": [
```

Comment:
@castastrophe - I think we should remove this. During the release for prerelease 27, we had to remove this from all of the components because we needed the dist directory to be published along with the src, demo, and test directories. 

I still use those other directories when I'm building apps. I use them as a reference for how to include the component. I also use the tests directories to see how events are working.

Reply:
I get that the other files are useful but they are referencable in the mono-repo.  We can always move the README with that same information into the files array for publication.

Reply:
Let's just update this to `/dist/<%= elementName %>.*js`

Reply:
We'll have to update the build task then - adding ./dist won't work for the way we're reading that value in for gulp

Reply:
Updated with the latest